### PR TITLE
auto-install: support dockerfile, intelephense, vetur, vim, yaml

### PR DESCRIPTION
**Pull request by kiennq**

### CODE REVIEW

File: lsp-dockerfile.el [hunk 1]
```diff
@@ -36,20 +36,29 @@
   :link '(url-link "https://github.com/rcjsuen/dockerfile-language-server-nodejs"))
 
 (defcustom lsp-dockerfile-language-server-command
-  '("docker-langserver" "--stdio")
+  '(nil "--stdio")
```
Comment (yyoncho) [inline lsp-dockerfile.el:39]: this being nil is hard to explain. Can you keep the previous value and change `(cl-first lsp-dockerfile-language-server-command)` to `(executable-find (cl-first lsp-dockerfile-language-server-command))` in the connection function?

Comment (kiennq) [reply lsp-dockerfile.el]: Done

File: lsp-vimscript.el [hunk 2]
```diff
@@ -31,7 +31,7 @@
   :group 'lsp-mode
   :link '(url-link "https://github.com/iamcco/vim-language-server"))
 
-(defcustom lsp-clients-vim-executable '("vim-language-server" "--stdio")
+(defcustom lsp-clients-vim-executable '(nil "--stdio")
```
Comment (yyoncho) [inline lsp-vimscript.el:34]: You have missed that place.

**Review by yyoncho — APPROVED** [review]

File: lsp-vimscript.el (hunk 2 above)
Comment (kiennq) [reply lsp-vimscript.el]: Thanks

**Pull request merged**